### DISCUSSION TITLE
Gowebly AUR in Arch Linux

### DISCUSSION THREAD

**Discussion by JGabrielGruber** [gowebly/gowebly#41] [Show and tell]
Hello!

I would like to inform that I've submitted the gowebly-bin to the ArchLinux User Repository!

https://aur.archlinux.org/packages/gowebly-bin

**Comment by koddr** [comment]
Hi, @JGabrielGruber

Thank you for that. If possible, I would be glad if you could add the possibility to add a binary to AUR when creating a release, e.g., with [GoReleaser](https://goreleaser.com/customization/aur/). I'm planning to do it myself, but in case you have time to do it 😉

**Comment by JGabrielGruber** [comment]
Hello @koddr,

Soon I'll be looking into that, sorry but I've got lost into some side projects; but yes, it's possible!
